Packs/DSPM/Integrations/DSPM/DSPM.py

```diff
@@ -0,0 +1,903 @@
+            "Cloud Environment": asset.get("cloudEnvironment", ""),
+            "Service Type": asset.get("serviceType", ""),
+            "Lifecycle": asset.get("lifecycle", ""),
+            "Open Risks Count": asset.get("openRisksCount", 0),
+            "Open Alerts Count": asset.get("openAlertsCount", 0),
+            "Encrypted": asset.get("encrypted", False),
+            "Open To World": asset.get("openToWorld", False),
+            "Tags": asset.get("tags", {}),
+            "Asset Dig Tags": asset.get("assetDigTags", []),
+        }
+        for asset in assets
+    ]
+    return parsed_assets
+
+
+def get_asset_details(client: Client, args: dict[str, Any]) -> CommandResults:
+    asset_id = args.get("asset_id", None)
+    if not asset_id:
+        raise ValueError("asset_id not specified")
+
+    asset_details = client.get_asset_details(asset_id)
+    demisto.debug(f"Asset details of : {asset_id}")
+    demisto.debug(asset_details)
+    return CommandResults(
+        outputs_prefix="DSPM.AssetDetails",
+        outputs_key_field="id",
+        outputs=asset_details,
+    )
+
+
+def get_asset_files_by_id(client: Client, args: dict[str, Any]) -> CommandResults:
+    asset_id = args.get("asset_id", None)
+    if not asset_id:
+        raise ValueError("Asset ID not specified")
+
+    page_number = 1
+    page_size = MAX_PAGE_SIZE  # You can adjust the page size as needed
+    all_files = []
+
+    while True:
+        params = {
+            "id": asset_id,
+            "page": page_number,
+            "size": page_size
+        }
+
+        # Fetch the asset files for the current page
+        response = client.get_asset_files(params)
+        files = response.get("files", [])
+
+        if not files:
+            break  # Stop when there are no more files to retrieve
+
+        # Append the files to the total list of files
+        all_files.extend(files)
+
+        # Increment page number for the next fetch
+        page_number += 1
+
+    files_count = len(all_files)
+
+    # Return the result without formatting the files structure
+    return CommandResults(
+        outputs_prefix="DSPM.AssetFiles",
+        outputs_key_field="filename",
+        outputs={"files": all_files, "filesCount": files_count},
+        readable_output=f"Total Files: {files_count}\nFiles: {all_files}"
+    )
+
+
+def get_data_types(client: Client) -> CommandResults:
+    """Command to fetch data types."""
+    data_types = client.get_data_types()
+    data_types_formatted = [
+        {"No": index + 1, "Key": dt} for index, dt in enumerate(data_types)
+    ]
+
+    if data_types_formatted:
+        readable_output = "### Data Types\n | No | Key  |\n |----|------|\n"
+        for item in data_types_formatted:
+            readable_output += f"| {item['No']}  | {item['Key']} |\n"
+    else:
+        readable_output = (
+            "### Data Types\n | No | Key |\n |----|-----|\n **No entries.**\n"
+        )
```

Comment:
please use tableToMarkdownm see this article:
https://xsoar.pan.dev/docs/integrations/code-conventions#tabletomarkdown

Reply:
Yes